integration-cli/docker_cli_run_test.go

```diff
@@ -477,6 +477,42 @@ func TestRunWithVolumesFromExited(t *testing.T) {
 	logDone("run - regression test for #4979 - volumes-from on exited container")
 }
 
+// Test create volume in a dirctory which is a symbolic link
+func TestRunCreateVolumesInSymlinkDir(t *testing.T) {
+	defer deleteAllContainers()
+	// This test has to create a file on host
+	hostFile := "/tmp/abcd"
```

Comment:
Ok, we need to use `iotuil.TempDir` here, and create a file in that dir using go instead of relying on `touch`.. and then a subsequent defer for `os.RemoveAll(tmpDir)`